executor/aggfuncs/aggfuncs.go

```diff
@@ -139,8 +139,21 @@ const (
 // to be any type.
 type PartialResult unsafe.Pointer
 
+// AggPartialResultMapper contains aggregate function results
+type AggPartialResultMapper map[string][]PartialResult
+
+type aggSpillHelper interface {
+	// SerializePartialResult will serialize meta data of aggregate function into bytes and put them into chunk.
+	SerializePartialResult(ctx sessionctx.Context, partialResult PartialResult, chk *chunk.Chunk, spillHelper *SpillSerializeHelper)
```

Comment:
Why not define this function as 
```suggestion
	SerializePartialResult(ctx sessionctx.Context, partialResults []PartialResult, chk *chunk.Chunk, spillHelper *SpillSerializeHelper)
```